spec/integration/puppet/gce_address_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require 'helpers/integration_spec_helper'
+
+describe "gce_address" do
+  it_behaves_like "a resource that can be created and destroyed" do
+    let(:type_name) { 'gce_address' }
+    let(:gcloud_resource_name) { 'addresses' }
+    let(:describe_args) { 'puppet-test-address --region us-central1' }
+    let(:expected_properties) { {'name'        => 'puppet-test-address',
```

Comment:
Having fixed names means multiple users running the tests at the same time on the same account will clash. This isn't as odd as it sounds as elsewhere we've running tests like these under a CI system which runs parallel tests across different versions of Puppet or host OSs. It would be useful to generate some level of unique names to support that usecase.